js/ui/popup.js

```diff
@@ -413,6 +416,18 @@ var Popup = Overlay.inherit({
             .toggleClass(POPUP_NORMAL_CLASS, !value);
     },
 
+    _shouldUseContentAutoResizableClass() {
+        if(this.option("height") !== "auto" || this.option("shading") || this.option("fullScreen") || this.option("showTitle") || this._getToolbarItems("bottom").length) {
+            return false;
+        }
+
+        return this.option("useAutoHeightWithLimits");
```

Comment:
We should write a test for each case that could be broken if we will remove this option. Maybe this will not be a breaking change and all cases are expected.